src/ctap/storage.rs

```diff
@@ -359,26 +367,23 @@ pub fn set_pin(
     pin_hash: &[u8; PIN_AUTH_LENGTH],
     pin_code_point_length: u8,
 ) -> Result<(), Ctap2StatusCode> {
-    let properties_key =
-        check_and_get_key_for_slot(env, slot_id, key::FIRST_PIN_PROPERTIES, key::PIN_PROPERTIES)?;
-    let mut value = env.store().find(key::FORCE_PIN_CHANGE)?.unwrap_or_default();
-    value.resize(env.customization().slot_count(), 0);
-    *(value
+    let mut force_pin_change_value = env.store().find(key::FORCE_PIN_CHANGE)?.unwrap_or_default();
+    force_pin_change_value.resize(env.customization().slot_count(), 0);
+    *(force_pin_change_value
         .get_mut(slot_id)
         .ok_or(Ctap2StatusCode::CTAP2_ERR_VENDOR_INTERNAL_ERROR)?) = 0;
```

Comment:
You are not using the `concat` interface for `FORCE_PIN_CHANGE` because it's simple enough to do it yourself?

Reply:
Yes, but if you prefer I use concat for consistency that's also ok.

Reply:
No strong opinion, so feel free to merge.

Reply:
I switched to using concatenated value, can you take a last look? Thanks